bluebottle/initiatives/serializers.py

```diff
@@ -158,6 +166,21 @@ class JSONAPIMeta:
         ]
         resource_name = 'initiatives'
 
+    def get_stats(self, obj):
+        activities = obj.activities.filter(status='succeeded')
+        stats = [activity.stats for activity in activities]
+        currency = properties.DEFAULT_CURRENCY
+
+        return {
+            'activities': len(activities),
+            'contributions': sum(stat['count'] for stat in stats),
+            'hours': sum(stat['hours'] or 0 for stat in stats if 'hours' in stat),
+            'amount': sum(
+                convert(Money(stat['amount']['amount'], stat['amount']['currency']), currency).amount
+                for stat in stats if 'amount' in stat
+            ),
+        }
```

Comment:
I guess we want to move the 'stats' calculation to the model if we want to show the complete stats in admin (or other locations) too. It's a bit unclear that we have different representations of 'stats' on model and serializer now.